sklearn/svm/classes.py

```diff
@@ -1140,8 +1140,20 @@ class OneClassSVM(BaseLibSVM, OutlierMixin):
         The offset is the opposite of `intercept_` and is provided for
         consistency with other outlier detection algorithms.
 
+    Examples
+    --------
+    >>> from sklearn.datasets import make_circles
+    >>> from sklearn.svm import OneClassSVM
+    >>> X, y = make_circles(n_samples=6, factor=0.4, random_state=41)
+    >>> clf = OneClassSVM(gamma='auto')
+    >>> clf.fit(X)
```

Comment:
please add:
```suggestion
    >>> clf.fit(X)   # doctest: +NORMALIZE_WHITESPACE
```

(Not very important but I'm not sure you're using the latest version)